src/mon/FSCommands.cc

```diff
@@ -213,6 +213,16 @@ class FsNewHandler : public FileSystemCommandHandler
       }
     }
 
+    const pool_stat_t *data_pool_stat = mon->mgrstatmon()->get_pool_stat(data);
+    if (data_pool_stat) {
+      int64_t data_num_objects = data_pool_stat->stats.sum.num_objects;
+      if (!force && data_num_objects > 0) {
```

Comment:
add `dout` that `--force` is overriding this check

Reply:
Done.